libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1505,7 +1510,7 @@ unsigned CompilerUtils::prepareMemoryStore(Type const& _type, bool _padToWords)
 
 	bool leftAligned = _type.category() == Type::Category::FixedBytes;
 
-	convertType(_type, _type, true);
+	convertType(_type, _type, _cleanup);
```

Comment:
If `_cleanup` is false, calling this function is not needed, is it?

Reply:
I think you're right. I'll add an `if`